<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemWordToolWdgClass</class>
 <widget class="QWidget" name="RemWordToolWdgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RemWordToolWdg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="leTransform"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnTransformFile">
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTransform">
        <property name="text">
         <string>转化</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="leImportPath"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnImportFile">
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnImport">
        <property name="text">
         <string>导入</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
